src/core/sock/sockinfo.h

```diff
@@ -328,6 +332,7 @@ class sockinfo : public socket_fd_api,
     uint32_t m_flow_tag_id; // Flow Tag for this socket
     bool m_rx_cq_wait_ctrl;
     uint8_t m_n_uc_ttl_hop_lim;
+    int m_bind_no_port;
```

Comment:
is it a boolean? if yes, make it `bool`.

Reply:
right, done.